src/test/java/io/github/ititus/math/number/BigComplexTests.java

```diff
@@ -0,0 +1,105 @@
+package io.github.ititus.math.number;
+
+import org.assertj.core.data.Offset;
+import org.junit.jupiter.api.Test;
+
+import static io.github.ititus.assertions.Assertions.assertThat;
+import static io.github.ititus.math.number.BigComplexConstants.ONE;
+import static io.github.ititus.math.number.BigComplexConstants.ZERO;
+import static io.github.ititus.math.number.BigRational.of;
+import static io.github.ititus.math.number.BigRationalConstants.*;
+import static org.assertj.core.api.Assertions.assertThatExceptionOfType;
+import static org.assertj.core.data.Offset.strictOffset;
+
+public class BigComplexTests {
+    public static final BigComplex ONE_I_TWO = BigComplex.of(BigRationalConstants.ONE, TWO);
+    public static final BigComplex TWO_I_TWO = BigComplex.of(TWO, TWO);
+    public static final BigRational TWICE_TWO_SQRT = TWO.sqrt().multiply(TWO);
+    public static final Offset<BigRational> EPSILON = strictOffset(of("0.001"));
+    public static final Offset<BigRational> STRICT_EPS = strictOffset(of("0.000001"));
+
+    @Test
+    public void test_one_real() {
+        assertThat(ONE.getReal()).isOne();
+    }
+
+    @Test
+    public void test_one_imag() {
+        assertThat(ONE.getImag()).isZero();
+    }
+
+    @Test
+    public void test_abs_squared() {
+        assertThat(TWO_I_TWO.absSquared()).isCloseTo(EIGHT, EPSILON);
+    }
+
+    @Test
+    public void test_abs() {
+        assertThat(TWO_I_TWO.abs()).isCloseTo(TWICE_TWO_SQRT, EPSILON);
+    }
+
+    @Test
+    public void test_angle() {
+        assertThat(TWO_I_TWO.angle()).isCloseTo(PI.divide(4), EPSILON);
+    }
+
+    @Test
+    public void test_negate() {
+        assertThat(TWO_I_TWO.negate()).isCloseTo(BigComplex.of(MINUS_TWO, MINUS_TWO), EPSILON);
```

Comment:
Please use a static import for `BigComplex.of` and remove the static import for `BigRational.of`